docs/getting-started.rst

```diff
@@ -12,9 +12,8 @@ Create a new official type of Remote Settings
 
 Basically, you will have to go through these 3 steps:
 
-1. Setup the `Mozilla VPN <https://mana.mozilla.org/wiki/display/IT/Mozilla+Corporate+VPN>`_ and request access to the server using `this Bugzilla template <https://bugzilla.mozilla.org/enter_bug.cgi?assigned_to=vpn-acl%40infra-ops.bugs&bug_file_loc=http%3A%2F%2F&bug_ignored=0&bug_severity=normal&bug_status=NEW&cc=mathieu%40mozilla.com&cc=wezhou%40mozilla.com&cf_fx_iteration=---&cf_fx_points=---&comment=Hello%2C%0D%0ACould%20you%20please%20add%20me%20%28LDAP%20user%40mozilla.com%29%20to%20the%20following%20groups%3A%0D%0A-%20%22vpn_kinto1_stage%22%0D%0A-%20%22vpn_kinto1_prod%22%0D%0Ain%20order%20to%20gain%20access%20the%20Remote%20Settings%20Admin%0D%0A%0D%0AThanks%21&component=Mozilla%20VPN%3A%20ACL%20requests&contenttypemethod=list&contenttypeselection=text%2Fplain&defined_groups=1&flag_type-4=X&flag_type-481=X&flag_type-607=X&flag_type-674=X&flag_type-800=X&flag_type-803=X&form_name=enter_bug&groups=mozilla-employee-confidential&maketemplate=Remember%20values%20as%20bookmarkable%20template&op_sys=Unspecified&priority=--&product=Infrastructure%20%26%20Operations&qa_contact=gcox%40mozilla.com&rep_platform=Unspecified&short_desc=VPN%20access%20for%20Remote%20Settings%20https%3A%2F%2Fsettings-writer.%2A.mozaws.net&target_milestone=---&version=Production>`_
```

Comment:
Maybe we want to keep the Mana link and the step to set up the VPN, and only remove the link to create a bug?

Reply:
Oh stupid me.